test/fixtures/command_line/bwb.xcodeproj/xcshareddata/xcschemes/BazelDependencies.xcscheme

```diff
@@ -9,17 +9,8 @@
          <ExecutionAction
             ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
             <ActionContent
-               title = "Set Bazel Build Output Groups"
-               scriptText = "if [[ -s &quot;$BAZEL_BUILD_OUTPUT_GROUPS_FILE&quot; ]]; then&#10;    rm &quot;$BAZEL_BUILD_OUTPUT_GROUPS_FILE&quot;&#10;fi&#10;">
-               <EnvironmentBuildable>
-                  <BuildableReference
-                     BuildableIdentifier = "primary"
-                     BlueprintIdentifier = "7E7D155EBCA520F35DEA3571"
-                     BuildableName = "BazelDependencies"
-                     BlueprintName = "BazelDependencies"
-                     ReferencedContainer = "container:test/fixtures/command_line/bwb.xcodeproj">
-                  </BuildableReference>
-               </EnvironmentBuildable>
+               title = "Clear Bazel Build Output Groups File"
+               scriptText = "if [[ -s &quot;$BAZEL_BUILD_OUTPUT_GROUPS_FILE&quot; ]]; then&#10;    rm &quot;$BAZEL_BUILD_OUTPUT_GROUPS_FILE&quot;&#10;fi">
             </ActionContent>
```

Comment:
@brentleyjones I separated the clear of the `$BAZEL_BUILD_OUTPUT_GROUPS_FILE` from the appending of the target IDs in the file. In doing so, the code that clears the `$BAZEL_BUILD_OUTPUT_GROUPS_FILE` no longer has an environment buildable set. From my tests, this seems OK. However, I just want to confirm that this seems reasonable to you.